[21874] Restyle WP details panel

mk84 opened the pull request:
https://community.openproject.org/work_packages/21874/activity

mk84 committed:
Fix description displayed only in one line on IE10

mk84 committed:
Show WP type and ID instead of description label on wp details panel

mk84 committed:
Show WP last updated date above description

mk84 committed:
Remove subline from wp details panel

mk84 committed:
Update styles for WP comments in details pane
* Use regular font style

mk84 committed:
Don't overlap editable descriptions

mk84 committed:
Remove border from wp description and align with title

mk84 committed:
Expand generic tables to 100%; remove hack to hide horizontal scrollbar

mk84 committed:
Fix line-height inside generic-tables

mk84 committed:
Ajust alignment of form elements inside tables

TeatroIO commented:
I've prepared a stage to preview changes. <a href='http://mk84-21874-restyle-wp-details.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://mk84-21874-restyle-wp-details.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

mk84 commented:
@furinvader I removed the subline from the work package details panel. But this had a wp watch button (a grey star). Can you please add a normal watch button like the one from fullscreen view and place it between the edit and more button in the bottom toolbar?

oliverguenther committed:
Split view: Add watch, unwatch buttons

oliverguenther committed:
Refactor watch functionality in ShowController

oliverguenther committed:
Fix unwatch function is Firefox

Firefox provides an `unwatch` property to watch and unwatch
(https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/Object/unwatch),
which will break a simple `!! workPackage.links.unwatch` check.

oliverguenther committed:
Feature spec

furinvader commented:
As the button is used on multiple locations, split and fullscreen view, it's best to move it in its own directive.

oliverguenther committed:
Rewrite as separate directive
